README.md

```diff
@@ -2,3 +2,4 @@
 [![Actions Status](https://github.com/AlexMomot-717/python-project-lvl1/workflows/hexlet-check/badge.svg)](https://github.com/AlexMomot-717/python-project-lvl1/actions)
 [![pre-commit hooks](https://github.com/AlexMomot-717/python-project-lvl1/workflows/pre-commit.yml/badge.svg)](https://github.com/AlexMomot-717/python-project-lvl1/actions)
 [![Maintainability](https://api.codeclimate.com/v1/badges/b902fd4658d626f88eeb/maintainability)](https://codeclimate.com/github/AlexMomot-717/python-project-lvl1/maintainability)
+https://asciinema.org/a/VF9daOkEFokjXQWJxdY4H52pJ
```

Comment:
please, use special markdown flavor to display it as clickable image (there should be a button somewhere on asciinema website)